library/general/src/lib/ui/text_helpers.rb

```diff
@@ -53,5 +53,17 @@ def wrap_text(text, wrap = 76, separator: " ", prepend_text: "",
 
       lines.join("\n")
     end
+
+    # Wrap a given text in direction markers
+    #
+    # @param [String] text to be wrapped
+    # @param [String] language code (it gets the current one by default)
+    # @return [String] wrapped text
```

Comment:
We should say (and test) that *text* may contain tags which will be passed through (not escaped)